packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.ts

```diff
@@ -0,0 +1,589 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
While I removed some (e.g: I renamed `_id` to `nextId`), others appeared to be a more dangerous change (e.g `attributes._cssText` here: https://github.com/DataDog/browser-sdk/pull/700/files#diff-1c7cef52f64a686ba523420f60390086b7a16d8f13b51ac8f0a3e79d90be827eR224)